src/OrchardCore.Modules/OrchardCore.Settings/SiteSettings.cs

```diff
@@ -1,26 +1,88 @@
+using System.Collections.Generic;
 using Microsoft.AspNetCore.Routing;
+using Newtonsoft.Json.Linq;
 using OrchardCore.Documents;
+using OrchardCore.Entities;
 
 namespace OrchardCore.Settings
 {
     // When updating class also update SiteSettingsDeploymentSource and SettingsStep.
     public class SiteSettings : DocumentEntity, ISite
     {
+        private JObject _properties = new();
```

Comment:
no a simple dictionary?

Reply:
The simple dictionary is below line 86

Here it is related to line 14 that defines a `new` Properties to override the `Entity.Properties`.